examples/phi2/user_script.py

```diff
@@ -211,8 +215,8 @@ def flatten_past_kv_inputs(past_key_values: List[Tuple[torch.Tensor, torch.Tenso
     past_kv = {}
     # Convert list of past_kv to dict of past_key and past_value
     for i, (past_k, past_v) in enumerate(past_key_values):
-        past_kv[f"past_key_values.{i}.key"] = past_k
-        past_kv[f"past_key_values.{i}.value"] = past_v
+        past_kv[f"past_key_{i}"] = past_k
+        past_kv[f"past_value_{i}"] = past_v
```

Comment:
Does the dynamo exporter change the name of the inputs? Or it is a result of the transformers optimization pass?

Reply:
it's the result of the transformers optimization pass